stories/analyses/Listing.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import Listing from "../../src/components/analyses/listing/Listing";
+import { mockAxios } from "../axiosMock";
+import { listing } from "./AnalysesMocks";
+import { UserProfileProvider } from "../../src/contexts/userProfile";
+export default {
+    title: "Analyses",
+};
+
+function ListingTest(props) {
+    return (
+        <UserProfileProvider>
```

Comment:
You shouldn't need this `UserProfileProvider` wrapper anymore.  It's in the storybook config (because I kept forgetting to add it to my stories and it was driving me crazy lol)

Reply:
Hmm jest test fails if remove the `<UserProfileProvider>`.

Analyses Listing view

    useUserProfile must be used within a UserProfileProvider

      12 |     const context = React.useContext(UserProfileContext);
      13 |     if (!context) {
    > 14 |         throw new Error(
         |               ^
      15 |             `useUserProfile must be used within a UserProfileProvider`
      16 |         );
      17 |     }

      at useUserProfile (src/contexts/userProfile.js:14:15)
      at Listing (src/components/analyses/listing/Listing.js:63:27)

Reply:
I think `data` jest passes because it is only testing table view. But am testing the whole listing in `analyses` jest. I will go ahead and modify the `analyses` jest to test only table view.